sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -220,31 +219,16 @@ object SharedState extends Logging {
   }
 
   /**
-   * Load hive-site.xml into hadoopConf and determine the warehouse path we want to use, based on
-   * the config from both hive and Spark SQL. Finally set the warehouse config value to sparkConf.
+   * Determine the warehouse path by spark conf, hadoop configuration and the initial options from
+   * the very first created SparkSession instance.
    */
-  def loadHiveConfFile(
+  def determineWarehouse(
```

Comment:
why the return type is still a `Map`?

Reply:
We should document what this method returns, as it's not clear from the name.

Reply:
make sense to me. the returned map is unrelated to this change，I will doc better